frontend/ui/elements/menu_activate.lua

```diff
@@ -2,55 +2,36 @@ local InfoMessage = require("ui/widget/infomessage")
 local UIManager = require("ui/uimanager")
 local _ = require("gettext")
 
-local function activateMenu() return G_reader_settings:readSetting("activate_menu") end
-
 return {
     text = _("Activate menu"),
     sub_item_table = {
         {
-            text = _("Swipe and tap"),
+            text = _("Tap"),
             checked_func = function()
-                local activate_menu = activateMenu()
-                if activate_menu == nil or activate_menu == "swipe_tap" then
-                    return true
-                else
-                    return false
-                end
+                return G_reader_settings:readSetting("activate_menu") ~= "swipe"
             end,
             callback = function()
-                G_reader_settings:saveSetting("activate_menu", "swipe_tap")
-                UIManager:show(InfoMessage:new{
-                    text = _("This will take effect on next restart."),
```

Comment:
Changes don't require a restart?

Reply:
There were 3 checkboxes, now remain only 2, so one InfoMessage is removed.